pgcli/packages/sqlcompletion.py

```diff
@@ -27,6 +28,20 @@ def suggest_type(full_text, text_before_cursor):
 
     identifier = None
 
+    def strip_named_query(txt):
+        """
+        This will strip "save named query" command in the beginning of the line:
+        '\ns zzz SELECT * FROM abc'   -> 'SELECT * FROM abc'
+        '  \ns zzz SELECT * FROM abc' -> 'SELECT * FROM abc'
+        """
+        pattern = re.compile(r'^\s*\\ns\s+[a-z0-9\-_]+\s+')
```

Comment:
One more minor update. 

```
pattern = re.compile(r'^\s*\\ns\s+[A-z0-9\-_]+\s+')
```

That should include all the alphabets uppercase and lowercase.